Integrations/integration-cyberark_AIM.yml

```diff
@@ -256,6 +256,13 @@ script:
             delete args.keywords;
             return getAccountDetails(args);
         default:
+            args.AppID = appid;
```

Comment:
if it's the default case you can delete the `fetch-incidents` one. nope?

Reply:
good point!

Reply:
@orius123 `fetch-credentials` has false on HumanReadable is it necessary?

Reply:
don't think so but need to validate.